Rectangle/lt.lproj/Main.strings

```diff
@@ -0,0 +1,749 @@
+
+/* Class = "NSTabViewItem"; label = "Settings"; ObjectID = "gtf-PD-IHm"; */
+"gtf-PD-IHm.label" = "Nustatymai";
+
+/* Class = "NSTabViewItem"; label = "Keyboard Shortcuts"; ObjectID = "uw2-9W-2jq"; */
+"uw2-9W-2jq.label" = "Karštieji klavišai";
+
+/* Class = "NSButtonCell"; title = "Restore window size when unsnapped"; ObjectID = "UZP-5q-D5Y"; */
+"UZP-5q-D5Y.title" = "Atkurti lango dydį po to, kai pašalinamas pririšimas";
+
+/* Class = "NSTextFieldCell"; title = "Gaps between windows"; ObjectID = "bg9-nw-YvU"; */
+"bg9-nw-YvU.title" = "Tarpai tarp langų";
+
+"Halves" = "Pusės";
+"Corners" = "Kampai";
+"Thirds" = "Trečdaliai";
+"Maximize" = "Maksimizuoti";
+"Size" = "Dydis";
+"Display" = "Ekranas";
+"Other" = "Kita";
+"Move to Edge" = "Perkelti į kraštą";
+"Fourths" = "Ketvirtys";
+"Sixths" = "Šeštosios";
+
+/* Class = "NSButtonCell"; title = "Import"; ObjectID = "RgZ-Jw-XQZ"; */
+"RgZ-Jw-XQZ.title" = "Importuoti";
+
+/* Class = "NSButtonCell"; title = "Export"; ObjectID = "IO3-Hi-7gC"; */
+"IO3-Hi-7gC.title" = "Eksportuoti";
+
+/* Class = "NSButtonCell"; title = "Move cursor along with window across displays"; ObjectID = "Pbz-DF-hgG"; */
+"Pbz-DF-hgG.title" = "Perkelti žymeklį kartu su langais tarp ekranų";
+
+/* Class = "NSMenuItem"; title = "cycle through displays"; ObjectID = "XlM-ch-cLG"; */
+"XlM-ch-cLG.title" = "perjungineti tarp ekranų";
+
+/* Class = "NSMenuItem"; title = "move to adjacent display on left or right"; ObjectID = "Z9d-Rl-RVq"; */
+"Z9d-Rl-RVq.title" = "pereiti prie gretimo ekrano kairėje arba dešinėje";
+
+/* Class = "NSMenuItem"; title = "do nothing"; ObjectID = "jww-Ju-S3d"; */
+"jww-Ju-S3d.title" = "nieko nedaryti";
+
+/* Class = "NSMenuItem"; title = "cycle ½, ⅔, and ⅓ on half actions"; ObjectID = "gHH-BV-5kP"; */
+"gHH-BV-5kP.title" = "perjungti tarp ½, ⅔ ir ⅓ pusėms";
+
+/* Class = "NSMenuItem"; title = "move to adjacent on left/right, or cycle size on half"; ObjectID = "3GE-la-fAZ"; */
+"3GE-la-fAZ.title" = "pereiti prie gretimo ekrano kairėje/dešinėje arba keisti pusės dydį";
+
+/* Class = "NSTextFieldCell"; title = "Repeated commands"; ObjectID = "2Zm-fl-PcC"; */
+"2Zm-fl-PcC.title" = "Kartojamos komandos";
+
+"Default Shortcuts" = "Numatytieji karštieji klavišai";
+"Currently using: " = "Dabar naudojama: ";
+"Cancel" = "Atšaukti";
+
+/* Class = "NSButtonCell"; title = "Show Todo Mode in menu"; ObjectID = "7yS-wj-uWD"; */
+"7yS-wj-uWD.title" = "Rodyti Todo režimą meniu";
+
+/* Class = "NSTextFieldCell"; title = "Todo application width"; ObjectID = "6e0-ji-qXw"; */
+"6e0-ji-qXw.title" = "Todo programos lango plotis";
+
+/* Class = "NSTextFieldCell"; title = "Reflow Todo shortcut"; ObjectID = "Fx0-sm-DrT"; */
+"Fx0-sm-DrT.title" = "Karštasis klavišas Todo lango padėties atnaujinimui";
+
+"Use frontmost.app as Todo App" = "Naudoti pirmame plane esantį langą kaip Todo programą";
+"Enable Todo Mode" = "Įjungti Todo programos režimą";
+"Perdėlioti Todo" = "Atnaujinti Todo lango padėtį";
```

Comment:
Looks like a few of the English keys got translated into Lithuanian, here and at the end. Would you mind making sure that the correct English keys are in? I'd give it a go myself, but sometimes these can move around between the different translation files and it's probably easier for you to check.